admin/sql/updates/2019-02-13-change-login-id-type.sql

```diff
@@ -0,0 +1,13 @@
+BEGIN;
+
+ALTER TABLE "user" DROP COLUMN user_login_id;
```

Comment:
is the idea that we'll run both updates in a row while releasing in prod?

Reply:
the first one is already run, i'll run this while releasing on beta for testing.